docs/pages/concepts/offloading/namespace-replication.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Namespace replication
+weight: 2
+---
+
+### The Liqo Namespace Model
+
+The Liqo resource replication model is composed of several concurring logics responsible for different kind of objects to replicate.
+One important example is represented by namespaces. 
+In particular, namespace replication across multiple clusters implements a key mechanism to extend seamleassly a cluster to some others.
+
+In the Liqo model, the namespace replication corresponds to a standard v1.Namespace and an associated [NamespaceOffloading](#) CR.
+The NamespaceOffloading resource allows to select the clusters which are the clusters where the application contained inside the namespace can be scheduled.
+
+In [a dedicated usage section](/usage/namespace_offloading), you can learn how to define and use a NamespaceOffloading resource activating the replication of the local namespace on peered clusters.
+
+In this section, you may discover how Liqo namespace model works under the hood, undestanding which resources and controllers are activated when a new namespaceOffloading is manipulated.
+
+#### Resources
+
+Liqo namespace replication involves instances of two kinds of CRDs:
+
+* First, the initial trigger is represented by the manipulation of a **NamespaceOffloading** (or the labelling of a namespace as explained in [the usage section](/usage/namespace_offloading)).
+  On the one hand, the *NamespaceOffloading* spec describes the properties of the replication, such as the name of the replicated namespaces.
+  On the other hand, the status collects the information about the actual status of remote namespaces (i.e. if the replication has succeeded).
+* Second, **NamespaceMap** resources contain the list of namespaces associated to a specific node.
+  The spec collects the list of desired namespaces for a specific remote cluster while the status updated information about their effective creation.
+  Each *NamespaceMap* can be alimented by several *NamespaceOffloading* instances that are targeting the same cluster for a remote namespace.
+  It is worth noting that the NamespaceMap status represent the source of truth to know the status of replicated namesapces on foreign clusters.
+
+#### Controllers
+
+The namespace replication logic is composed of several distinct controllers, responsible for different aspects of namespace controllers:
+
+* **NamespaceOffloading Controller**: processes the NamespaceOffloading spec and outputs namespace entries to NamespaceMap specs.
+* **NamespaceMap Controller**: enforces the namespace creation of the remote namespaces on remote clusters and updates the status of the NamespaceMaps.
+* **OffloadingStatus Controller**: updates the status of the NamespaceOffloading resources by reading the NamespaceMaps statuses.
+
+#### Workflow
+
+In the next figure, you can observe a representation of the overall workflow steps of the replication process:
+
+![](/images/namespace-replication/replication.png)
```

Comment:
This picture is very good, but is very difficult to understand without an example.
I would suggest to duplicate this picture: a first part contains the "theory", such as now;a second part contains a simple, very simple example, which could be referenced in the explanation steps below.

Reply:
I added an example figure, let me know if it looks good to you.